src/index.js

```diff
@@ -22,6 +34,16 @@ export {
     Statement,
     OverviewTable,
     TabPanel,
+    handleChange, 
+    handleSelect, 
+    handleMultiSelect,
+    prepareForSave, 
+    isValid, 
+    addSelectLists,
+    isInput,
+    isSelect,
+    isMultiSelect,
     EntityDetail,
```

Comment:
Ich weiß ehrlich gesagt nicht so richtig, ob das da so rein passt. Es sind teilweise sehr spezifische Anwendungsfunktionen, die meiner Meinung nach nicht in so ein Package passen... 
Hatte ich die übersehen irgendwann? Oder sind die einfach rein gerutscht?

Reply:
ich kann auch nur addSelectLists exportieren. Wenn du eine bessere Lösung weißt...vielleicht ist es dann aber schlauer einen separaten PR zu machen.

Reply:
Ich möchte EntityDetails und ein paar Helfer-Funktionen einfach nicht in jedem generierten Projekt warten müssen. Es gibt noch speziefische Klassen, die dann im Projekt selbst angepasst werden.